### ISSUE TITLE
[NPC] Azuregos - Add missing path

### ISSUE THREAD

**Issue by Rushor** [dalaranwow/dalaran-wow#4470]
http://wotlk.openwow.com/npc=6109

Azuregos should walk around in Azshara

* source: http://wowwiki.wikia.com/wiki/Azuregos

**Comment by Rushor** [comment]
DB/Creature: Add pathing for Azuregos in Azshara

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4470
* commit: 38f8226ec4d50c35a20066cd407f48bf5782bc9c

![GitHub Logo](http://i.imgur.com/bkFyEK5.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Teridan-dal-wow** [comment]
Think the same can be said for Spirit of Azuregos. From wowwiki:

> To open the gates of Ahn'Qiraj, players must at one stage speak with Azuregos. As Azuregos was likely to be killed within a few hours of spawning, this would have made the quest difficult, if not impossible, to complete. To avoid this a Spirit of Azuregos can always be found wandering through southern Azshara. The Spirit cannot be attacked or killed, drops no loot, and is purely for players on the blue shard line of the scepter quest. You can speak to the Spirit, but if you are not on the proper stage of the quest, he says nothing interesting.

**Comment by Rushor** [comment]
@Teridan-dal-wow already done https://github.com/dalaranwow/dalaran-wow/issues/4459 :)
